app/views/layouts/application.html.erb

```diff
@@ -18,8 +18,12 @@
                 aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
           <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <%if user_signed_in? %>
```

Comment:
having the methods in the controller is more practical, in the comments above you could apply something similar

Reply:
Yes sir